### PR TITLE
Export `easystats_packages()`

### LINKED ISSUE #411: Export `packages_on_cran()`
Currently, there is no easy way for users/developers to programmatically access a list of core `{easystats}` packages. 

Exposing this internal function would be helpful in this regard.

### PR DESCRIPTION
**Pull request by etiennebacher**
Close #411 

I also removed `.packages_on_github()` since it was equivalent to `.packages_on_cran()` and I think the likelihood of having another package (on Github only) is quite low

### CODE REVIEW

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]
Overall, LGTM. Thanks for working on this!

- Can you please also add a simple test? 
Maybe checking the length and the contents of the vector.

- Update pkgdown config file to also include this new function

File: R/utils.R [hunk 1]
```diff
@@ -42,98 +42,53 @@
...
       stringsAsFactors = FALSE,
       row.names = NULL
     )
-
-    .add_easystats_dev_pkgs(out, easystats_not_on_cran)
-  }
-}
-
-
-.add_easystats_dev_pkgs <- function(out, easystats_not_on_cran) {
```
Comment (IndrajeetPatil) [inline R/utils.R:85]: Yeah, I agree about getting rid of this. 

The only reason I had been hesitant to get rid of it thus far was `{circus}`, but that package is not really of any value or relevance to our users anyway.

**Comment by strengejacke** [comment]
> I also removed .packages_on_github() since it was equivalent to .packages_on_cran() and I think the likelihood of having another package (on Github only) is quite low

Yes, this function had historical reasons, when not all packages were on CRAN. It's obsolete now.

**Review by strengejacke — APPROVED** [review]

**Comment by etiennebacher** [comment]
Thanks for the review

> Can you please also add a simple test?

I don't see the point, this function just prints a hardcoded character vector, it's not like some internal code can modify its output.

> Update pkgdown config file to also include this new function

Thanks, I forgot about it. Done

**Comment by IndrajeetPatil** [comment]
In that case, can you ignore directive for code coverage for the body of the function?

**Review by IndrajeetPatil — APPROVED** [review]

**Comment by etiennebacher** [comment]
Any idea where this error comes from?

https://github.com/easystats/easystats/actions/runs/9438885978/job/25996361636?pr=417#step:6:68

**Comment by etiennebacher** [comment]
Looks like this is related to `Matrix`: https://stackoverflow.com/a/77504843

Anyway, not related to this PR so merging

**Pull request merged**